internals/models.py

```diff
@@ -885,6 +885,46 @@ def getSortingMilestone(feature):
 
     return allowed_feature_list
 
+  @classmethod
+  def get_in_milestone(
+      self, version=None, show_unlisted=False, milestone=None):
+    if milestone == None:
+      return None
+
+    logging.info('Getting chronological feature list in milestone %d', milestone)
+    q = Feature.query()
+    q = q.order(Feature.name)
+    q = q.filter(Feature.shipped_milestone == milestone)
+    desktop_shipping_features = q.fetch(None)
+
+    # Features with an android shipping milestone but no desktop milestone.
+    q = Feature.query()
+    q = q.order(Feature.name)
+    q = q.filter(Feature.shipped_android_milestone == milestone)
+    q = q.filter(Feature.shipped_milestone == None)
+    android_only_shipping_features = q.fetch(None)
+
+    # Constructor the proper ordering.
+    all_features = []
+    all_features.extend(desktop_shipping_features)
+    all_features.extend(android_only_shipping_features)
+
+    # First sort by name, then sort by feature milestone (latest first).
```

Comment:
No longer sorting by milestone.

Reply:
Oops... I forgot to make changes to this comment. 

Done. Thanks